src/web/collectors/CollectorRestartButton.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+
+import CollectorsActions from 'collectors/CollectorsActions';
+
+const CollectorsRestartButton = React.createClass({
+  propTypes: {
+    collector: React.PropTypes.object.isRequired,
+    backend: React.PropTypes.string.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      disabled: false,
+    };
+  },
+
+  componentDidMount() {
+    this.interval = setInterval(this._refreshButtonState, this.BUTTON_REFRESH);
```

Comment:
We need to be careful to only render this component on a page with a limited amount of collectors, otherwise we can create many parallel requests to the server.
If we need to show the button for many collectors at the same time we should think about implementing a bulk call to get the button states.

I consider this optional if we only show it for a single collector right now.